plugins/CoreAdminHome/Controller.php

```diff
@@ -118,7 +118,7 @@ public function generalSettings()
             '' => '',
             'Plain' => 'Plain',
             'Login' => 'Login',
-            'Crammd5' => 'Crammd5',
+            'Cram-md5' => 'Cram-md5',
```

Comment:
not sure how this is used... does this affect any config setting and does anything need to migrated maybe?

Reply:
will add a simple migration